galsim/wfirst/wfirst_psfs.py

```diff
@@ -161,7 +215,10 @@ def getPSF(SCAs=None, approximate_struts=False, n_waves=None, extra_aberrations=
 
             # In this case, we need to pad the edges of the pupil plane image, so we cannot just use
             # the stored file.
-            tmp_pupil_plane_im = galsim.fits.read(galsim.wfirst.pupil_plane_file)
+            if pupil_plane_type == 'long':
+                tmp_pupil_plane_im = galsim.fits.read(galsim.wfirst.pupil_plane_file_longwave)
```

Comment:
Not covered.  This is the high_accuracy one, so probably don't want to actually run this in the travis runs.  Maybe can use the same trick to do an assert_raises check that will hit this along the way.